io.github.vinser.pixyne.yml

```diff
@@ -0,0 +1,34 @@
+app-id: io.github.vinser.pixyne
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.golang
+command: pixyne
+
+finish-args:
+    - --share=ipc 
+    - --socket=x11
+    - --device=dri
```

Comment:
doesn' fyne support wayland?

Reply:
fyne does not support wayland yet

Reply:
I did not find any explicit statement about fyne support for wayland, only mentions in bug fixes. I myself cannot check whether the application will work on wayland since my host on which I develop the app is Ubuntu Mate X11 Odroid N2+ SBC.
The contributor of fyne in manifest of his fyne based app [flathub io.github.jacalz.rymdport.yml ](https://flathub.org/apps/io.github.jacalz.rymdport) give such a comment in:

> #-  The below are commented out until there is out of the box support for wayland.
> #- --socket=fallback-x11 # Only create the x11 socket if wayland isn't avaliable.
> #- --socket=wayland

So I must not do this